tests/queries/0_stateless/01544_file_engine_settings.sh

```diff
@@ -2,6 +2,7 @@
 set -eu
 
 CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
```

Comment:
Having to add this to every file would be incovenient...

From their docs here: https://github.com/koalaman/shellcheck/wiki/SC1090
> ShellCheck v0.7.2+ will strip a single expansion followed by a slash, e.g. ${var}/util.sh or $(dirname "${BASH_SOURCE[0]}")/util.sh, and treat them as ./util.sh. This allowing the use of source-path directives or -P flags to specify the a location.

I only have 0.7 so it doesn't work. Maybe we can use the new version in the CI and avoid all these directives?

Reply:
On the other hand, we have to copy the `CURDIR` thing anyway, so copying three lines instead of two doesn't change much.

Reply:
I have initially tested with the new version, and can confirm it is working file.